python/protobuf.c

```diff
@@ -253,18 +253,24 @@ static const char *PyUpb_GetClassName(PyType_Spec *spec) {
 }
 
 PyTypeObject *PyUpb_AddClass(PyObject *m, PyType_Spec *spec) {
-  PyObject *type = PyType_FromSpec(spec);
+  PyObject *type = (void*)PyType_FromSpec(spec);
```

Comment:
you added the `(void*)` here but didn't change the lhs of the object.  I am guessing you don't need this cast.

Reply:
Done.